data-node/src/main/java/org/graylog/datanode/Configuration.java

```diff
@@ -74,19 +74,19 @@ public class Configuration extends BaseConfiguration {
     private boolean disableNativeSystemStatsCollector = false;
 
     @Parameter(value = "opensearch_location")
-    private String opensearchLocation = "./data-node/bin/opensearch-2.4.1";
+    private String opensearchLocation = "./bin/opensearch-2.4.1";
```

Comment:
This should be `dist/opensearch-2.4.1` to match the change we did in `misc/datanode.conf` earlier.

```suggestion
    private String opensearchLocation = "dist/opensearch-2.4.1";
```